clients/admin-ui/src/features/dataset/DatasetTable.tsx

```diff
@@ -10,15 +10,13 @@ import { Dataset, GenerateTypes } from "~/types/api";
 
 import {
   setActiveDatasetFidesKey,
-  useGetAllFilteredDatasetsQuery,
+  useGetAllDatasetsQuery,
 } from "./dataset.slice";
 
-const DatasetsTable = () => {
+const DatasetTable = () => {
   const dispatch = useDispatch();
   const router = useRouter();
-  const { data: datasets } = useGetAllFilteredDatasetsQuery({
-    onlyUnlinkedDatasets: false,
```

Comment:
Did the requirements change on this? I think we wanted this behavior when shipping this initially. Or was this filtering option accidentally placed where it shouldn't be?

What problem is this solving?

Reply:
We ran into an issue where we wanted to be able to edit the data category for a dataset created from SaaS connector template being instantiated. Do you remember a reason why we wouldn't want to return those datasets in this table?

Reply:
I do, the reason was we wanted to prevent users from editing SaaS connector template datasets 😅 At the time our (you and I) big push was to not let users change the datasets that are linked to SaaS template based connectors. Or not display the option to them in the UI rather. 

After that we established that, we talked about how some parts of a SaaS dataset are okay to modify but not all of them. Since that UI, AFAIK, doesn't limit a user to only editing the "safe" properties of a SaaS dataset we decided to prevent them from doing it all together until the UI was updated to restricting users to editing the "safe" properties for SaaS datasets.

Assuming, this is the same area of the codebase that I'm thinking it is and we're okay with users have the ability to potentially corrupt a SaaS template dataset then this should be good to ship. Otherwise the we'd need to update the UI to specifically handle SaaS datasets.